digits/dataset/tasks/create_db.py

```diff
@@ -81,6 +85,10 @@ def __setstate__(self, state):
                 delattr(self, 'encode')
             else:
                 self.encoding = 'none'
+        if self.pickver_task_createdb <= 3:
+            print 'Upgrading CreateDbTask to version 4'
+            # nothing to do, version 4 adds field create_db_log_file
+            # but this field is ignored if None
```

Comment:
Do we really need a version bump here? Seems to me like we can make `create_db_log_file` an optional attribute for version 3.